core/src/main/resources/properties/item_dao.xml

```diff
@@ -112,7 +112,7 @@
 	</query>
 	<query>
 		<name>findAllItemsByVersionId</name>
-		<sql>select i.* from versioning_map v, item i 
+		<sql>select distinct i.* from versioning_map v, item i 
```

Comment:
Due to recent changes to crf_version and  form_layout , where we use the same crf_version against all the from_layouts within CRF, caused the versioning_map table to have multiple [item and crf_version combination] records for each form_layout within CRF.  For this reason, the page was loading duplicate records. By adding distinct to the sql, I'm able to get unique records.